### PR TITLE
NFC Push Tx feature

### PR DESCRIPTION
**Pull request by doc-hex**
This feature allows single-tap broadcast of the freshly-signed transaction.

Once enabled with a URL, the COLDCARD will show the NFC animation
after signing the transaction. When the user taps their phone, the
phone will see an NFC tag with URL inside. That URL contains the
signed transaction ready to go, and once opening in the mobile
browser, that URL will load. The landing page will connect to a
Bitcoin node (or similar) and send the transaction on the public
Bitcoin network.

### CODE REVIEW

File: docs/nfc-pushtx.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# NFC Push Tx
+
+This feature allows single-tap broadcast of the freshly-signed transaction.
+
```
Comment (nvk) [inline docs/nfc-pushtx.md:4]: Add here:

PSBT ==[SD|QR|NFC]==> COLDCARD signed TXN ==[NFC tap]==> Phone Browser ==> Server TXN Broadcast

File: shared/auth.py [hunk 2]
```diff
@@ -806,9 +806,20 @@ async def interact(self):
         except BaseException as exc:
             return await self.failure("PSBT output failed", exc)
 
-        from glob import NFC
+        from glob import NFC, settings
 
         if self.do_finalize and txid and not hsm_active:
+
+            # if NFC PushTx is enabled, do that w/o questions.
```
Comment (scgbckbone) [inline shared/auth.py:813]: meh for starting without confirmation of action. Would be much better inside the loop on line 826. This way users can export multiple times on possible failure

**Review by scgbckbone — COMMENTED** [review]
should also have an `NFC Tools` entry to read signed txn from microSD and share PushTx url

File: shared/auth.py (hunk 2 above)
Comment (doc-hex) [reply shared/auth.py:813]: disagree. they enabled it so they want it. already loops for multiple attempts when the tap fails / phone craps out, etc.

**Comment by doc-hex** [comment]
> should also have an `NFC Tools` entry to read signed txn from microSD and share PushTx url

yes, good idea ... will add

**Pull request merged**